express/server.js

```diff
@@ -77,8 +77,9 @@ Sentry.init({
   ],
   tracesSampleRate: 1.0,
   tracesSampler: samplingContext => {
-    // sample out transactions for http OPTIONS requests
-    if (samplingContext.request.method == 'OPTIONS') {
+    // sample out transactions from http OPTIONS requests hitting endpoints
+    const request = samplingContext.request
+    if (request && request.method == 'OPTIONS') {
```

Comment:
sometimes there was not 'request' property on the samplingContext. I'm not quite sure why, but this fixed. Still getting the expected transactions coming in and nothing extra.